src/main/java/com/bynder/sdk/service/asset/AssetService.java

```diff
@@ -35,9 +35,10 @@
     /**
      * Get Tags.
      *
+     * @param tagQuery Filter resulting tags
      * @return {@link Observable} with list of {@link Tag}.
      */
-    Observable<Response<List<Tag>>> getTags();
+    Observable<Response<List<Tag>>> getTags(TagQuery tagQuery);
```

Comment:
Shouldn't we keep the old one as well to not break code that uses the sdk? (not sure how much we care about compatibility in this new FS release)

Reply:
passing in an empty tagQuery should be an easy enough migration